collectors/apps.plugin/apps_plugin.c

```diff
@@ -4247,7 +4247,10 @@ int main(int argc, char **argv) {
         normalize_utilization(apps_groups_root_target);
 
         send_resource_usage_to_netdata(dt);
+
+#ifndef __FreeBSD__
```

Comment:
Can we make it Linux only? (not FreeBSD -> is Linux). Does it make sense?

Reply:
based on the current implementation of apps.plugin, it seems that not FreeBSD means Linux. I haven't checked the flags used during build though